helm-chart/build.py

```diff
@@ -115,7 +121,38 @@ def publish_pages():
     subprocess.check_call(
         ['git', 'push', 'origin', 'gh-pages'],
         cwd='gh-pages',
-        env=dict(os.environ, GIT_SSH_COMMAND='ssh -i ../travis')
+        env=dict(os.environ, GIT_SSH_COMMAND=f'ssh -i ../{HELM_CHART_DEPLOY_KEY_NAME}')
+    )
+
+def update_mybinder_deployment(version):
+    commit = last_git_modified('.')
+    subprocess.check_call([
+        'git', 'clone',
+        'https://github.com/jupyterhub/mybinder.org-deploy', 'staging'],
+        env=dict(os.environ, GIT_SSH_COMMAND='ssh -i {MYBINDER_DEPLOY_KEY_NAME}')
+    )
+    subprocess.check_call(['git', 'checkout', 'staging'], cwd='staging')
+    subprocess.check_call(['git', 'remote', 'add', 'meeseeks', 'ssh://git@github.com/MeeseeksBox/mybinder.org-deploy'])
+
+    with open(os.path.join('staging', 'config', 'common.yaml')) as f:
+        values = yaml.load(f)
+
+    values['version'] = version
+
+    with open(os.path.join('staging', 'config', 'common.yaml')) as f:
+        yaml.dump(values, f)
+
+
+    subprocess.check_call(['git', 'add', '.'], cwd='staging')
```

Comment:
git add . is very dangerous always, no? especially with possible secrets around. I'd rather we explicitly only added the two files we have changed.